docs/namespaces-per-cluster.md

```diff
@@ -0,0 +1,97 @@
+# Namespaces per cluster Workload
+
+The Namespaces per cluster workload playbook is `workloads/cluster-limits-namespaces-per-cluster.yml` and will run the namespaces per cluter limits workload on your cluster.
```

Comment:
I think this segment of the above sentence "and will run the namespaces per cluter limits workload on your cluster." should be changed to "and will run the namespaces per cluster workload on your cluster."  What do you think?

Reply:
Fixed it, thanks.